### PR TITLE
Fix #13: Expand Security and Privacy Considerations

### LINKED ISSUE #13: Security and Privacy considerations for ALS
I would suggest updating security/privacy considerations with the following:

> Ambient Light Sensor API provides information about lighting conditions near the device environment. There are several potential privacy risks related with exposing this information on the web.
> - Information leaks about the user’s surrounding and work habits.
> - Profiling. Readout from  Ambient Light Sensor can potentially induce information leaks about the user’s interests, web use and work habits, as well as the users’ surrounding. This information might be used to enhance the user profiling and behavioral analysis.
> - Cross-device linking and tracking. Access to sufficiently precise readouts of Ambient Light sensors potentially enhance cross-device linking techniques. Such situation may arise if two different devices access web sites including same third-party scripts that compare lighting levels over time. 
> - Cross-device communication. Verbose readout of Ambient Light Sensor could be applied to receive messages emitted by other devices in nearby location. A simple messaging method could arise by multiple devices flashing their screens or camera LEDs and reading out responses with Ambient Light Sensors
> 
> The user agent SHOULD expose minimized lighting levels. 
> 
> A verbose, high-resolution readout from the API SHOULD be subject to permissions.
> 
> The user agent SHOULD inform the user about the current and past uses of the API.

### PR DESCRIPTION
**Pull request by anssiko**
PTAL @lknik @alexshalamov @pozdnyakov

***
[Preview](https://s3.amazonaws.com/pr-preview/w3c/ambient-light/security-privacy.html) | [Diff](https://s3.amazonaws.com/pr-preview/w3c/ambient-light/eacd638...77ebda1.html)

### CODE REVIEW

File: index.bs [hunk 1]
```diff
@@ -71,8 +74,32 @@ Examples {#examples}
...
+    time.
+  - Cross-device communication. Verbose readout of Ambient Light Sensor could
+    be applied to receive messages emitted by other devices in nearby
+    location. A simple messaging method could arise by multiple devices
+    flashing their screens or camera LEDs and reading out responses with
+    Ambient Light Sensors.
+
+To mitigate these Ambient Light Sensor specific threats, user agents should
+use the following mitigation strategies:
+  - <a>limit maximum sampling frequency</a>, and
```
Comment (alexshalamov) [inline index.bs]: should it be "and or"? At the moment, it sounds that US should use both simultaneously.

File: index.bs [hunk 2]
```diff
@@ -71,8 +74,32 @@ Examples {#examples}
...
 ===================================
 
-There are no specific security and privacy considerations
-beyond those described in the Generic Sensor API [[!GENERIC-SENSOR]].
+Ambient Light Sensor provides information about lighting conditions near
+the device environment. Potential privacy risks include:
+
+  - Information leaks about the user’s surrounding and work habits.
+  - Profiling. Readout from Ambient Light Sensor can potentially induce
+    information leaks about the user’s interests, web use and work
```
Comment (alexshalamov) [inline index.bs]: _"web use"_  - Is there such term?
_"work habits"_ - was in previous bullet.
_"Information leaks  about .... users’ surrounding"_  - was in previous bullet.

File: index.bs (hunk 1 above)
Comment (anssiko) [inline index.bs]: See https://github.com/w3c/ambient-light/pull/33/commits/6cb119ec768bdb9f82dcaffb4600c282aaf16a84

File: index.bs (hunk 2 above)
Comment (anssiko) [inline index.bs]: I'll let @lknik comment on this text (a copy-paste from https://github.com/w3c/ambient-light/issues/13#issue-174271517 with some very minor copyediting).

Comment (lknik) [inline index.bs]: Hello indeed those were too elaborate. Feel free to simplify, i.e.

"work habits" -> "use patterns"

Profiing - you are right, unnecessary repeat.

**Comment by lknik** [comment]
Included a new comment at #13

**Comment by anssiko** [comment]
Updated the PR based on feedback, PTAL.

**Comment by anssiko** [comment]
@lknik, I added cross-origin in the profiling risk. Now we should be covered, thanks!

File: index.bs [hunk 3]
```diff
@@ -78,8 +78,8 @@ Ambient Light Sensor provides information about lighting conditions near
 the device environment. Potential privacy risks include:
 
   - Profiling. Ambient Light Sensor can leak information about user's use
-    patterns and surrounding. This information can be used to enhance user
-    profiling and behavioral analysis.
+    patterns and surrounding cross-origin. This information can be used to
```
Comment (lknik) [inline index.bs]: Not sure about "surrounding cross-origin"? The surrounding in my mind here was the physical environment ;)

Comment (anssiko) [inline index.bs]: I defer to a native speaker to give me a grammar lesson.

"Can leak information ... cross-origin."

Sounds good to me.

Comment (lknik) [inline index.bs]: It's not about the language ;) Cross-origin sounds like we're speaking about web/origins. I'm speaking about surrounding, like, even the physical environment.

Comment (anssiko) [inline index.bs]: Please give us a concrete suggestion. A complete sentence.

Comment (anssiko) [inline index.bs]: @lknik, I'm trying to understand what is the concrete suggestion you are trying to make. Are you suggesting we remote the word "cross-origin", i.e. revert commit https://github.com/w3c/ambient-light/commit/77ebda1947379ef6ec98838e0fa97e9677cf20d2 ?

Comment (lknik) [inline index.bs]: Yeah, revert. It ("merge") mangled a few things

Comment (anssiko) [inline index.bs]: Thanks, done.

**Pull request merged**